tiflash/tiflash-configuration.md

```diff
@@ -183,6 +183,12 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2.0 引入，代表 PageStorage V3 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
+    dt_storage_blob_heavy_gc_valid_rate = 0.5
+
+    ## 从 v6.2.0 引入，PageStorage V3 数据文件中的数据块会按照此参数对齐 IO 边界。默认值为 0，代表不进行 IO 边界对齐.
```

Comment:
Looks like this setting is not useful. I think we don't need to mention it here.

Reply:
removed.